packages/cds-odata-annotation-converter/tsconfig.json

```diff
@@ -1,33 +1,34 @@
 {
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "incremental": true,
-    "outDir": "dist",
-    "rootDir": "src",
-    "lib": ["ES2021"]
-  },
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "incremental": true,
+        "outDir": "dist",
+        "rootDir": "src",
+        "lib": ["ES2021"],
+        "target": "ES2021"
```

Comment:
This one still has target set to `ES2021`. Was it intentional?

Reply:
<img width="1464" alt="Screenshot 2024-07-05 at 15 40 31" src="https://github.com/SAP/open-ux-tools/assets/10102054/cccc6ac0-1bb7-443d-80ac-3b57c3e24fc4">
Tests fail when updated in annotation packages. 
All seem to show this line as https://github.com/SAP/open-ux-tools/blob/main/packages/cds-annotation-parser/src/parser/parser.ts#L345

Reply:
ok, it looks like there is some issue with chevrotain with that target. We can look at it separately.